### ISSUE TITLE
Way to play online?

### ISSUE THREAD

**Issue by XxsjrapsxX** [sd805/l4d2vr#52]
is there a way to play online if you host a server or something? I would love to play in vr with friends who dont live with me but idk if thats possible

**Comment by Richleth** [comment]
From the readme: Multiplayer (only if the host has the mod and the server is set to local)

This does NOT mean LAN as in you have to be on the same network, this means the server itself is hosted locally on your machine. If you want to play with friends either they have to be using the mod and hosting or you have to host. Whoever is host would just have to invite the other friends to the game lobby. 

Just make sure to either make sure everyone has the same workshop mods installed or sv_consistency is set to 0.

On the other note the Flatscreen to VR Modding Community Discord has a l4d section that includes an LFG section.

**Comment by SGTPinchaloaf** [comment]
Yes you can play online.....only on a private server with the VR mod installed.

**Comment by Joshiloveyou** [comment]
> Yes you can play online.....only on a private server with the VR mod installed.

Do you know of any servers with the mod installed?

**Comment by T-Rizzle12** [comment]
> > Yes you can play online.....only on a private server with the VR mod installed.
> 
> Do you know of any servers with the mod installed?

You can create your by creating a lobby and change the server from Official Dedicated to Local Server. This will host the lobby on your computer instead of a valve server. Just make sure that sv_consistency is set to 0 and in multiplayer options and set download custom content to None.
